dist/marble.js.LICENSE.txt

```diff
@@ -45,7 +45,1201 @@
  * @license MIT
  */
 
+/*! ../../../global/settings */
```

Comment:
I'm not sure what all this cruft is? 🤔  Could you just delete this file and rebuild dist?